src/studio/nuxeo-studio/jsf-web-ui-migration.md

```diff
@@ -0,0 +1,110 @@
+---
+title: 'How to migrate your JSF UI configuration to Web UI'
+description:
+tree_item_index: 985
+review:
+  comment: ''
+  date: '2018-10-15'
+  status: ok
+toc: true
+---
+
+{{#> callout type='info' heading='part of early access program'}}
+JSF UI to Web UI migration tool is available through our early access program, offering stable features you can co-create with us as they are only partially complete.<br /><br />
+
+- Not a member of it yet? [Learn about its benefits, what's planned and how to get in]({{page page="early-access-program"}}).
+<br /><br />
+- Already a member? Help us co-creating it by <a href="https://docs.google.com/forms/d/e/1FAIpQLSdoDfpJOnFwkxHAHOk5aq5KHh9myxF6fISugnMzVmvuQSp-fw/viewform?usp=pp_url" target="_blank">telling us about your experience</a>.
+{{/callout}}
+
+The JSF UI to Web UI migration tool is a safe and automated way to create an equivalent Web UI configuration based on your existing JSF UI configuration.
+
+## What Value Does it Bring?
+The migration tool dramatically reduces the effort needed to transition to Web UI, our user interface that makes use of modern web standard technologies to provide an exceptional user experience.
+
+It has been designed to be:
+
+### Safe
+No data is deleted when the migration tool is run. If you are not satisfied with the result, you can:
+- go back to the state your project was before triggering it anytime,
+- delete the additional configuration that was generated partially or totally.
+
+### Informative
+After a migration run, a detailed report is created so that you can check the results.
+
+### Fast
+A complete run typically finishes in a couple of minutes or less.
+
+## How Does it Work?
+### Launching Migration
+Steps below assume early access is activated on your project. If it's not yet, you can have a look at [how to activate it]({{page page="early-access-program"}}).
+
+1. Checkout the branch you want to migrate in `Source Control -> Branch Management` screen. If your project does not benefit from branch management, you can skip this step.
+2. Go to the `Configuration -> Migration Services` screen.
+3. Press the `start migration` button and confirm.
+
+Note that if you are using the intermediate or advanced commit modes, you need to discard or commit and push any work in progress before the migration can be launched.
+
+{{#> callout type='info' heading='migration conditions'}}
+Your branch can be migrated if it fulfills the following conditions:
+1. Target Nuxeo Server version is LTS 2017
+1. Your dependencies include the JSF UI package
```

Comment:
`1.` -> `2.` 🙂

Reply:
It's the proper way to write a numbered list on the doc so that if one day you want to add a new step in between existing ones, you don't have to care about the numbers, it's calculated automatically.